app/boards/arm/nrfmicro/nrfmicro_11.dts

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (c) 2020 Okke Formsma, joric
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+/dts-v1/;
+#include <nordic/nrf52840_qiaa.dtsi>
+#include "arduino_pro_micro_pins.dtsi"
+
+/ {
+	model = "nrfmicro_flipped";
+	compatible = "joric,nrfmicro";
+
+	chosen {
+		zephyr,code-partition = &code_partition;
+		// zephyr,console = &uart0;
+		//zephyr,bt-mon-uart = &uart0;
+		//zephyr,bt-c2h-uart = &uart0;
```

Comment:
Are these lines kept for future reference and commented ? If not, we could remove it.